packages/nodes-base/credentials/ContentfulDeliveryApi.credentials.ts

```diff
@@ -0,0 +1,31 @@
+import { ICredentialType, NodePropertyTypes } from 'n8n-workflow';
+
+export class ContentfulDeliveryApi implements ICredentialType {
+	name = 'contentfulDeliveryApi';
+	displayName = 'Delivery API';
+	properties = [
+		{
+			displayName: 'Space Id',
+			name: 'space_id',
+			type: 'string' as NodePropertyTypes,
+			default: '',
+			required: true,
+			description: 'The id for the Cotentful space.'
```

Comment:
The id for the Cotentful space. 👉 The id for the Contentful space.

The n is missing

Reply:
👍